### DISCUSSION TITLE
彭载舟 | 致全国同胞的一封信

### DISCUSSION THREAD

**Discussion by xijinping0** [The-Run-Philosophy-Organization/run#583] [Ideas]
https://yibaochina.com/?p=248021

![口号](https://user-images.githubusercontent.com/111333464/195966724-5fa8c6d0-5f95-4602-85e6-48986464b28c.jpg)

全国军人武警同胞们:

全国警察同胞们:

全国各行各业的同胞们:

今天，中国正处在一个百年未有之大变局的时代，上一次大变局还是1915年12月12日，袁世凯复辟封建帝制，推翻共和，宣布接受帝位。同样的事情发生在2018年3月11日，习近平操纵的全国人大代表在枪口下投下了屈辱的选票，将宪法中的主席任期进行了清零，改成后患无穷的终身制。

同胞们，从两千多年封建帝制结束的1912年，到实现共和110年后的今天，从1978年改革开放44年后的今日，中国人民重新站在了十字路口。如果我们沉默，纵容习近平搞独裁终身制，那么我们的一切都可能灰飞烟灭，重新回到改革开放前，再一次穿越到封建王朝，我们的财富将在内耗中清零，我们的自由将在内斗中送进牢笼。如果进行宪政改革，阻止习近平非法连任，我们在经济上继续突飞猛进，跨过中等收入陷阱，政治上我们将进入文明和谐的自由民主社会，每个人都不必人人自危，不必看着独裁者战战兢兢的苟且。一个自由、民主、繁荣、富强的新中国是每个中国人的向往和愿望。为实现这一目标，离不开全国所有人民的努力，更需要人民子弟兵和警察的正义支持。因为中国是十四亿人的中国，不是独裁者的私有财产，解放军是保家卫国的人民军队，不是习近平的打手，警察是保护人民生命财产安全的人民警察，不是习近平的保安。

习近平口口声声为了人民，实际只是欺骗百姓的谎言，他的一切目的就是实现自己的皇帝梦领袖梦，这就是他的中国梦。他已经不能代表国家，已经不能代表共产党，已经不能代表人民，从宪法清零的那一天起，作为国家主席、党的总书记、军委主席已经完全失去合法性。习近平不仅从法律上失去了作为一国之主的资格，从能力上也不具备治国理政的水平。看看过去十年的表现。

无耻无德。作为一个没有经过人民通过选票授权的，仅仅因为幸运捡到了主席印玺的接班人，吃着前几任领导积累的老本，就像一个富二代，带着大金链子纹着身，恐吓人民威胁全世界，整天吹着自己的牛皮，在经济建设上没有任何贡献，却把中国社会推进中等收入陷阱，在政治上开历史倒车，以袁世凯为导师，做着不穿黄袍只穿透明西装的皇帝梦，大搞个人崇拜，无德无能却把自己伪装成人民领袖。丘吉尔曾说过，当领袖站起来，人民必然跪下去，事实的确如此。从规定政治局委员需要向他述职，到要求军队和各级领导对他绝对忠诚，从网络删帖封号，到残暴打击各种异见人士，无不显示他那不受约束的权力的任性。显然，习近平册封自己为人民领袖和军队统帅，仰仗着军队的枪杆子，把子弟兵变成了自己的打手，把警察和武警当成了自己的保安。

无知无能。从用左手敬礼，到念稿错字连篇，从各种前后矛盾的经济政策，再到毫无逻辑的政治思想，从空洞乏味的宣传口号，再到违反科学和人性的防疫措施，所有这些无不显示习近平的无知。自上台那一天起，开始各种折腾，以产业升级之名暴力驱赶所谓的低端产业低端人口，以环保之名关停多种实体企业，以共同富裕之名打击民营经济，用纳税人的钱财搞雄安新区，搞一带一路，高价引进低端留学生，各种外国撒币砸钱买好评，又因为修宪搞独裁与发达民主国家对抗，面临经济脱钩和产业链断裂危险，所有这一切，百姓没有得到任何实惠，大中型企业裁员工人失业，小微企业倒闭小老板破产，外贸订单流失，政府债台高筑国库空虚，偷摸印钞发债抢劫百姓财富等等，现在又以防疫之名将百姓关进方舱集中营，不但洗劫财富还绑架人身自由。为何如此折腾百姓，只有两个原因，一是学识浅薄能力平庸，根本不懂现代市场经济运行规律，社会运行法则，既自负又无能，正所谓心比天高，命比纸薄。二是为了在二十大上获得非法连任，为穿上皇帝的新装，以北朝鲜为榜样，金正恩为导师，正悄悄的将国门关闭，将百姓赶进圈里，让百姓失去自由，幻想人民失去反抗机会和能力，强行承认非法连任的合法性。但是他想错了，人民不会让独裁者如愿，人民军队更不会让习近平得逞。

对于当今的中国，安全威胁已经完全不是外敌，中国已经是世界第二大经济体，有榜眼级的军事实力，天上有卫星战机，海上有航母潜艇，既有洲际导弹又有核武器，对任何外敌都有另对方瑟瑟发抖的威慑力量。当习近平忽悠我们说，欧美列强侵略中国奴役中国百姓时，绝对是在为自己的治国无能找借口，为自己想要非法连任转移视线。作为对比，看看习近平眼中的榜样国家朝鲜，作为一个资源匮乏和拥有核武器的国家，根本不存在外敌入侵，真正侵略朝鲜人民利益的是以金氏家族为首的权贵阶级，奴役朝鲜人民的既不是南韩也不是美帝，就是他们的所谓领袖奴隶主金正恩。

在中国古代，每个封建王朝到最后亡朝，都是因为帝王昏庸无能，官僚腐败巧取豪夺，百姓民不聊生引起。从指鹿为马的秦二世到傻皇帝司马衷，从好大喜功的杨广到腐朽保守的慈禧。对于他们来说，外敌入侵只是加速了他们灭亡的进程。历史上的五胡乱华和八国联军入侵，并不是仅仅因为外敌强悍，最主要的是他们自己腐化无能。正是因为晋惠帝司马衷的愚钝无力，才引起八王之乱，又因为八王之乱才导致国力衰弱，最后引来五胡乱华。同样，正是因为闭关锁国的清王朝腐朽傲慢，思想上不思进取，才导致国力军力衰弱，但是嘴巴又极具战狼精神，昏庸慈禧可以同时向十一个列强宣战，最后导致八国联军侵华，进而间接导致清朝灭亡。而同时期的日本，经过明治维新改革，学习西方现代文明，迅速成为世界强国，曾经的中国小弟，俨然成了随意捏拿中国的大哥。对于当今的中国来说，威胁人民利益的不是外敌而是内贼，真正的侵略是腐败，真正的压迫是专制，真正的奴役是独裁。所以，反腐败就是反侵略，争取投票权就是反压迫，阻止独裁国贼习近平非法连任就是救中国。

在这里，我们想对全国同胞们说：

军人同胞们，你们是人民军队，不是独裁者的打手。

警察同胞们，你们是人民警察，不是独裁者的保安。

党员同胞们，你们是共产党的主人，不是独裁者的党奴，不要做独裁者的帮凶。

媒体同胞们，你们的责任是监督政府传播普世价值，不是为权贵唱赞歌的喉舌。

失业破产的同胞们，你们本可以自食其力，不应成为习近平治国无能的牺牲品。

企业家同胞们，你们是创造价值的纳税人，不是任权贵宰割的唐僧肉。

公务员同胞们，你们是为人民服务的公仆，不是独裁者的监工。

大学生同胞们，你们是祖国的未来，不是独裁者培养的家丁。

知识分子同胞们，你们是传播科学文化知识的园丁，不是为独裁者洗脑的工具。

全体同胞们，我们是中华人民共和国的主人，如果我们沉默，那么我们所有人，都将成为独裁者的奴隶。为了可以投票竞选，为公平正义，为自由民主，为我们自己，更为我们子孙后代的幸福，让我们来一场新的护国运动。我们可以罢课和平抗议，我们可以罢工和平抗议，我们要一气呵成，罢免独裁者习近平，我们要直接选举一个没有谎言、受公民监督、可以被弹劾的新主席，我们必须生活在一个没有奴役的社会里，让我们一起行动起来吧，走向天安门，走进中南海！

在此强烈呼吁军人和警察同胞们，希望你们像蔡锷、李烈钧、唐继尧等讨伐窃国大盗袁世凯一样勇敢，不要助纣为虐，不要为独裁者卖命，不要为独裁者当炮灰。做为一名军人，效忠的是宪法，是全体公民，并不是某个党或者某个人。如果对党效忠，那就是党卫军，如果对个人效忠，那就是打手。当今世界，任何一个民主国家的军队都不会效忠于一个党派或者个人，试想，如果美国军队只对民主党效忠，意味着民主党指挥枪，那么他们就会立刻把枪口对准共和党，马上变成一党专制社会，既然民主党的枪杆子可以对准共和党，也就可以对准任何其他反对党，也就可以对准任何持不同政见的其他公民，那么美国的民主瞬间消失。再如果，美国军队效忠拜登，那么美国瞬间就变成了拜登一人独裁社会。同样，韩国总统轮流换，军队效忠宪法永不变。正是因为朝鲜军队绝对效忠于金正恩，军队必然充当了金正恩的打手，才有了金氏家族世袭奴役朝鲜人民的力量。

在106年前护国运动中的主角–蔡锷将军，已经在道德与道义中的做出了正确选择，蔡锷将军是所有军人的榜样，在国家大义和个人情感之间他勇敢的选择了国家大义。

家境贫寒的蔡锷赴日本留学，袁世凯拿出一千大洋为蔡锷助学，显然有恩于蔡锷。当袁世凯促成清王朝交出王权，和平进入共和政体，蔡锷对袁世凯更是敬佩有加，同样袁世凯对蔡锷也非常器重，给予各种高官厚禄。当1915年12月12日，袁世凯复辟封建帝制，推翻共和的时候，蔡锷将军毅然决然的对有知遇之恩的袁世凯发动了护国战争，在袁世凯做了83天皇帝梦后，黯然退位。可以看出，蔡锷在关乎国体国运的道义面前具有坚定的人格底线，在讲滴水之恩当涌泉相报的传统道德观念中，蔡锷在精神上没受到任何束缚。正是这种正义凛然的行为，使得蔡锷将军流芳百世，实现了伟大称帝中国梦的窃国大盗袁世凯却遗臭万年!

今天，中国的每一个将军，每一个战士，你们又回到了像当年蔡锷讨伐袁世凯的历史抉择中，只是由窃国大盗袁世凯变成了做着现代版皇帝梦的窃国贼习近平。

今天，我们最强烈的呼吁人民子弟兵同胞们，武警同胞们，警察同胞们，当底层百姓为了生活为了工作走上街头游行时，当人民群众为了公平正义走上街头抗议时，当高校师生为了自由民主走上街头罢课时，当全国所有同胞为了反独裁反专制走上街头行动时，希望你们站在全国同胞一边，不要听从独裁者的命令，不要服从镇压者的指挥，当任何人下达逮捕和镇压平民的命令时，他就变成了杀人犯和刽子手，无论你是普通士兵，还是各级军官，都有权反抗，有权反击，不会承担任何法律责任。你们是人民子弟兵，不是独裁者的打手，你们的枪口应该对准外敌，对准侵略者，不是对准自己的同胞，自己的兄弟姐妹，你们要像当年蔡锷讨伐袁世凯一样，勇敢的加入人民的队伍，加入全中国人民进行的新护国运动，为建立自由民主繁荣富强的新中国而行动。

致全国同胞的一封信

中华人民共和国全民普选委员会

中国共产党自由竞选委员会

附录1：

[下载《罢工 罢课 罢免独裁国贼习近平》](https://yibaochina.com/download/depose-xijinping.pdf)

附录2：议报收到的邮件

Hi，各位朋友们，非常抱歉，这是一封群发信，没有时间尊呼您的大名，因为我们马上要行动，不想被国安特务跟踪发现。附件内容是倒习的方案和攻略，同时也是一篇讨伐国贼习近平的檄文以及治国理政的解决方案。我们的基本方法是罢课、罢工、汽车鸣笛等非暴力合法的抗议，我们首先要点燃自由的星星之火，然后引起燎原之势，由你我大家共同参与，最后达到中南海内斗，我们期望出现像蔡锷一样的将军，一举推翻独裁政权。各位同胞们，无论你身在国外还是国内，在未来一到两周甚至更长时间，希望你们昼夜不停的加入战斗。如果您身在海外，希望您多多发帖，让更多的军人、媒体、大学生等各个群体参与其中。如果您身在国内，希望您能勇敢的走上街头，如果有特殊的困难，希望你们多多发帖参与信息战。各位热爱自由的同胞们朋友们，留给我们的时间不多了，改变中国命运的时刻就要到了，为了阻止国贼习近平的非法连任，让我们一起行动起来，我们都将变成时代的英雄，我们也将流芳百世!

![汽车鸣笛](https://user-images.githubusercontent.com/111333464/195966766-55f76a7b-bab5-48b5-81f0-e97117481c34.jpg)
![罢工](https://user-images.githubusercontent.com/111333464/195966772-25ffdccf-db6d-471f-bf45-d3bf045b521f.jpg)
![罢课](https://user-images.githubusercontent.com/111333464/195966775-a4ddfca9-7928-4bf5-a1c2-c7fd2af78ca9.jpg)
![起义](https://user-images.githubusercontent.com/111333464/195966776-bdfd3966-b0cf-4a4b-9ad6-dbdfb86c992a.jpg)

【议报首发，转载请加上出处和链接：https://yibaochina.com/?p=248021】

**Comment by leon-fudan** [comment]
哈哈，回头一看，帖子很搞笑吧。

**Reply by lance111l** [reply]
小编只是一个傻瓜瞎叫
